app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+import abc
+from app.players.player import Player
+
+
+class Dwarf(Player, abc.ABC):
```

Comment:
Just import ABC class from abc module. The same in all other similar cases

Reply:
You mean exact abc instead of abc.ABC?

Reply:
I literally see zero sense in your suggestions xD
Make according to readme... -> created 3 abstract classes.
No! Make according to readme -> inherited from abc -> abc.ABC
No, import ABC from ABC.
Why? For what purpose? Why it bother definition? Have it changes code, or visability, or stile, or anything?
I clearly can't get a thing.

Reply:
But I still question why I should do this to both classes inherited from abstract Player.
Explicit over implicit?